src/Symfony/Bridge/Doctrine/SchemaListener/MessengerTransportDoctrineSchemaSubscriber.php

```diff
@@ -11,18 +11,22 @@
 
 namespace Symfony\Bridge\Doctrine\SchemaListener;
 
+use Doctrine\Common\EventSubscriber;
 use Doctrine\DBAL\Event\SchemaCreateTableEventArgs;
 use Doctrine\DBAL\Events;
 use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+use Doctrine\ORM\Tools\ToolEvents;
 use Symfony\Component\Messenger\Bridge\Doctrine\Transport\DoctrineTransport;
 use Symfony\Component\Messenger\Transport\TransportInterface;
 
 /**
  * Automatically adds any required database tables to the Doctrine Schema.
  *
  * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @deprecated since Symfony 6.3
  */
-final class MessengerTransportDoctrineSchemaSubscriber extends AbstractSchemaSubscriber
+final class MessengerTransportDoctrineSchemaSubscriber extends MessengerTransportDoctrineSchemaListener implements EventSubscriber
```

Comment:
we should also remove the methods from the class, relying on the parent implementations when possible (same in other similar classes)

Reply:
correct!